datasets/us/medicare_exclusions/iowa/crawler.py

```diff
@@ -0,0 +1,121 @@
+from typing import Dict, Generator
+from rigour.mime.types import XLSX
+from openpyxl import load_workbook
+from normality import slugify, stringify
+from datetime import datetime
+import openpyxl
+
+from zavod import Context, helpers as h
+
+
+def parse_sheet(
+    context: Context,
+    sheet: openpyxl.worksheet.worksheet.Worksheet,
+    skiprows: int = 0,
+) -> Generator[dict, None, None]:
+    headers = None
+
+    row_counter = 0
+
+    for row in sheet.iter_rows():
+        # Increment row counter
+        row_counter += 1
+
+        # Skip the desired number of rows
+        if row_counter <= skiprows:
+            continue
+        cells = [c.value for c in row]
+        if headers is None:
+            headers = []
+            for idx, cell in enumerate(cells):
+                if cell is None:
+                    cell = f"column_{idx}"
+                headers.append(slugify(cell))
+            continue
+
+        record = {}
+        for header, value in zip(headers, cells):
+            if isinstance(value, datetime):
+                value = value.date()
+            record[header] = stringify(value)
+        if len(record) == 0:
+            continue
+        yield record
+
+
+def crawl_item(row: Dict[str, str], context: Context):
+
+    enrollment_type = row.pop("enrollment-type")
+
+    if enrollment_type is None:
+        return
+
+    if enrollment_type == "Individual":
+        first_name = row.pop("first-name")
+        last_name = row.pop("last-name")
+        entity = context.make("Person")
+        entity.id = context.make_id(first_name, last_name)
+        entity.add("firstName", first_name)
+        entity.add("lastName", last_name)
```

Comment:
Always use h.apply_name to add name parts to make sure the entity ends up with a `name` property constructed in a standardised way https://zavod.opensanctions.org/best_practices/merge_checklist/#:~:text=all%20Persons%20have%20a%20name%20property.%20Add%20first%20and%20last%20name%20via%20helpers.apply_name.